fiftyone/utils/eval/coco.py

```diff
@@ -292,6 +296,7 @@ def __init__(
         )
 
         self._classwise_AP = np.mean(precision, axis=(0, 2))
+        self._classwise_AR = np.mean(recall_sweep, axis=0)
```

Comment:
@brimoor I am using `config.compute_mAP` for mAR since they are closely related. Perhaps the flag should be renamed.

Reply:
The naming is less than ideal, but for backwards compatibility we should probably not rename it.